server/ws/index.js

```diff
@@ -36,13 +36,17 @@ let accumulatedVolume = 0;
 let accumulatedLargeTransactionHashes = [];
 
 // https://github.com/cryptocode/nano-websocket-sample-nodejs/blob/master/index.js
-const ws = new ReconnectingWebSocket("wss://www.nanolooker.com/ws", [], {
-  WebSocket: WS,
-  connectionTimeout: 10000,
-  maxRetries: 100000,
-  maxReconnectionDelay: 2000,
-  minReconnectionDelay: 10,
-});
+const ws = new ReconnectingWebSocket(
+  process.env.REACT_APP_WS_DOMAIN || "wss://www.nanolooker.com/ws",
```

Comment:
Explained better 😄 it has to do with this:

need to use "REACT_APP_" so we can use it directly on the client side
"REACT_APP_" is just a prefix that React uses to avoid accidental secret leaks
https://create-react-app.dev/docs/adding-custom-environment-variables/